FEXCore/Source/Utils/CPUInfo.cpp

```diff
@@ -14,17 +15,31 @@
 namespace FEXCore::CPUInfo {
 #ifndef _WIN32
 uint32_t CalculateNumberOfCPUs() {
-  char Tmp[PATH_MAX];
+  static uint32_t NumCPUs {};
```

Comment:
IIRC `uint32_t` writes are atomic on arm64 anyway, but might as well be explicit that it must be so for thread-safety:
```suggestion
  static std::atomic<uint32_t> NumCPUs {};
```

Reply:
Switched to atomic